packages/reporter/cypress/e2e/unit/formatted_message.cy.ts

```diff
@@ -0,0 +1,30 @@
+import { formattedMessage } from '../../../src/commands/command'
+
+describe('formattedMessage', () => {
+  it('returns empty string when message is falsy', () => {
+    const result = formattedMessage('')
+
+    expect(result).to.equal('')
+  })
+
+  it('maintains special characters when using "to match"', () => {
+    const specialMessage = 'expected **__*abcdef*__** to match /__.*abcdef.*__/'
+    const result = formattedMessage(specialMessage)
+
+    expect(result).to.equal('expected <strong><strong><em>abcdef</em></strong></strong>  to match <strong>/__.*abcdef.*__/</strong>')
```

Comment:
Looks like the E2E tests are failing because of an extra space character that came along for the ride 👽 this should fix it! 
```suggestion
    expect(result).to.equal('expected <strong><strong><em>abcdef</em></strong></strong> to match <strong>/__.*abcdef.*__/</strong>')
```